.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
+
+out/*
```

Comment:
No, that's fine. You can leave it like this and then the person creating the Maven project adds the line.